vmm/sandbox/src/sandbox.rs

```diff
@@ -220,6 +223,8 @@ where
         sandbox.start().await?;
         let sandbox_clone = sandbox_mutex.clone();
         monitor(sandbox_clone);
+        // sync clock
+        sandbox.sync_clock().await;
```

Comment:
maybe we should call it in the post_start(), so that for some hypervisor, maybe the time is synchronized automatically and we don't need to do the sync here.